src/video/resources/uploads.js

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Mux Assets
+ * Copyright(c) 2018 Mux Inc.
+ */
+const Base = require('../../base');
+
+/**
+ * @private Base asset path for the Mux API
+ * */
+const PATH = '/video/v1/uploads';
+
+/**
+ * @private
+ * Build the base asset path for the Mux API
+ * */
+const buildBasePath = uploadId => `${PATH}/${uploadId}`;
+
+/**
+ * Uploads Class - Provides access to the Mux Video Uploads API
+ *
+ * @example
+ * const { Video } = new Mux(accessToken, secret);
+ *
+ * // Create an upload
+ * Video.Uploads.create({ new_asset_settings: { playback_policy: 'public' } });
+ */
+class Uploads extends Base {
+  /**
+   * Creates a direct upload with the specified JSON parameters
+   * @extends Base
+   * @param {Object} params - Upload JSON parameters (e.g timeout)
+   * @returns {Promise} - Returns a resolved Promise with a response from the Mux API
+   *
+   * @example
+   * const muxClient = new Mux(accessToken, secret);
+   * const { Video } = muxClient;
+   *
+   * // Create a new upload
+   * const upload = await Video.Uploads.create({new_asset_settings: {playback_policy: 'public'}});
+   * // Now push a file to the URL returned.
+   * fs.createReadStream(pathToFile).pipe(request.put(upload.url))
+   *
+   * @see https://docs.mux.com/reference#upload-an-asset
+   */
+  create(params) {
+    if (!params) {
+      return Promise.reject(new Error('Params are required for creating a direct upload'));
+    }
+
+    return this.http.post(PATH, params);
+  }
+
+  /**
+   * Deletes an upload
+   * @param {string} uploadId - The ID for the upload intended for deletion
+   * @returns {Promise} - Returns a resolved Promise with a response from the Mux API
+   *
+   * @example
+   * const { Video } = new Mux(accessToken, secret);
+   *
+   * // Delete an upload
+   * Video.Uploads.remove(uploadId);
```

Comment:
Do we use `remove` elsewhere for delete things? Like... why not `delete?

Cool if we already use remove elsewhere, just asking

Reply:
Yeah, all the other modules used `remove`.

Reply:
`delete` is a reserved word in Javascript and i believe we were using `deleteAsset` or `removeAsset` before, but there was a request to keep it to one word for consistency.